camera_alignment_core/tests/test_alignment_core.py

```diff
@@ -107,14 +113,20 @@ def test_generate_alignment_matrix_reproducability(self):
         pixel_size_xy = optical_control_image.physical_pixel_sizes.X
 
         # Act
-        (alignment_matrix_1, _,) = generate_alignment_matrix(
+        (
+            alignment_matrix_1,
+            _,
+        ) = generate_alignment_matrix(
             optical_control_image_data,
             reference_channel=2,  # TaRFP
             shift_channel=3,  # CMDRP
             magnification=magnification,
             px_size_xy=pixel_size_xy,
         )
-        (alignment_matrix_2, _,) = generate_alignment_matrix(
+        (
```

Comment:
Is your linter set up the same way we default to?  I think there is a standard, like maybe a make target for the repo?